packages/editor/editor.css

```diff
@@ -167,7 +167,7 @@
 	--color-text: hsl(0, 0%, 0%);
 	--color-text-0: hsl(0, 0%, 11%);
 	--color-text-1: hsl(0, 0%, 18%);
-	--color-text-3: hsl(220, 2%, 65%);
+	--color-text-3: hsl(204, 4%, 45%);
```

Comment:
@steveruizok this tweaks the colors to match `--tla-color-text-3` which is the right contrast.

one knock-on side-effect is that MinimapManager piggybacks off of the original color. So, the minimap shape color is affected by this. I think it looks fine but lemme know if you think we need a separate color for it.